fmriprep/interfaces/tests/test_images.py

```diff
@@ -0,0 +1,53 @@
+import time
+import numpy as np
+import nibabel as nb
+from nipype.interfaces import nilearn as nl
+from .. import images as im
+
+import pytest
+
+
+@pytest.mark.parametrize('nvols, nmasks, ext, factor', [
+    (1000, 10, '.nii', 2),
+    (1000, 10, '.nii.gz', 5),
```

Comment:
these parameters triggered a `MemoryError`. Maybe 500 would be enough for these tests?

Reply:
The goal was to seriously stress, but I guess too much. It works with 500, as long as we're happy we're testing enough TRs that way.